packages/react-components/react-accordion/stories/Accordion/AccordionSizes.stories.tsx

```diff
@@ -5,7 +5,7 @@ export const Sizes = () => (
   <>
     <Accordion collapsible>
       <AccordionItem value="1">
-        <AccordionHeader size="small">Accordion Header 1</AccordionHeader>
+        <AccordionHeader size="small">Small Header</AccordionHeader>
         <AccordionPanel>
           <div>Accordion Panel 1</div>
```

Comment:
Does the numeral one in the panel help in the explanation? This numeral one is repeated in the other panels as well. Is this digit needed? 
Maybe it only needs to read `Accordion Panel`?